src/services/UserService.ts

```diff
@@ -105,7 +106,11 @@ export class UserService {
 			throw new APIError(HttpCode.BadRequest, AuthenticateError.AccountNotFound);
 		}
 
-		const user = await getRepository(User).findOne({ email });
+		const user = await getRepository(User)
+			.createQueryBuilder('user').where('user.email= :email', { email })
+			.addSelect('user.password')
+			.getOne();
+
```

Comment:
Could you use this instead? It just means other user relations get loaded by default:

```js
const user = await getRepository(User).findOne({
	where: {
		email
	},
	select: ['password']
});
```

Reply:
Tried it now but that doesn't work.  I've attached a screenshot of the error I received from npm test. 
![image](https://user-images.githubusercontent.com/43858192/91489447-c2e29380-e8a8-11ea-9997-e37a0d876188.png)

Reply:
Does the query builder not load by default? I assumed it was SELECT * WHERE user.email = email?

Reply:
In Typeorm, if you exclude the password field using `@Column({ select: false })` you have two options:
- Define every column you want to select in the `.findOne()` method call (incl. the password field)
- Use the `QueryBuilder` with `addSelect`